### PR TITLE
III-4045 UI improvements to match mockups for availability of single calendar type

### PR DESCRIPTION
**Pull request by LucWollants**
### Changed
- Make radio button group label optional
- Add title component for status and booking on the single calendar page

---

Ticket: https://jira.uitdatabank.be/browse/III-4045

### CODE REVIEW

**Review by simon-debruijn — COMMENTED** [review]
These changes look good. But first we should fix https://github.com/cultuurnet/udb3-frontend/pull/311

File: src/ui/RadioButtonGroup.js [hunk 1]
```diff
@@ -15,7 +15,9 @@ const RadioButtonGroup = ({
 }) => {
   return (
     <Stack className={className} as="div" spacing={3} {...getStackProps(props)}>
-      {groupLabel && <Label variant={LabelVariants.BOLD}>{groupLabel}</Label>}
+      {groupLabel ? (
+        <Label variant={LabelVariants.BOLD}>{groupLabel}</Label>
+      ) : null}
```
Comment (simon-debruijn) [inline src/ui/RadioButtonGroup.js:20]:
```suggestion
      {groupLabel && <Label variant={LabelVariants.BOLD}>{groupLabel}</Label>}
```
Btw this code would also work, as it was. Conditions resulting in false are not rendered by React

Comment (LucWollants) [reply src/ui/RadioButtonGroup.js:20]: I tried that approach first but then I got this error
```
Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.
```

Comment (geoffreydhuyvetters) [reply src/ui/RadioButtonGroup.js:20]: It's because it can return `false`, which a Stack/Inline can't handle

**Review by geoffreydhuyvetters — APPROVED** [review]
Nice one 💪

**Comment by LucWollants** [comment]
> Nice one 💪
That's because of the nice help during the pair programming 🙏

**Pull request merged**